miRge.pl

```diff
@@ -36,7 +36,7 @@
 my $version = "2.0";
 local $ENV{PATH} = "$ENV{PATH}:".$miRgePath;
 $miRgePath =~ s/\/[^\/]+\.pl/\//;
-my $refPath = $miRgePath."miRge.seqLibs/";
+my $refPath = $miRgePath."usr/local/miRge/miRge.seqLibs/";
```

Comment:
This was needed due to the directory structure within http://atlas.pathology.jhu.edu/baras/miRge/miRge.seqLibs.tar.gz. mirge.seqLibs is not at the top of that structure. Based on the README instructions to un-tar that file at the top level of miRge/, that path should be correct.